.github/workflows/documentation.yml

```diff
@@ -151,7 +151,7 @@ jobs:
 
       - name: Build Sphinx documentation
         run: |
-          pip install sphinx sphinx-sitemap
+          pip install sphinx sphinx-sitemap sphinx-material
```

Comment:
Pin the versions here? Or in a separate PR.

Reply:
We could add it to the requirements.txt file.